subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_resolution.adoc

```diff
@@ -248,6 +261,37 @@ If multiple incompatible Gradle versions are in play, all should be used when se
 | `modules-2`           | `files-2.1`           | `metadata-2.90`           | Gradle 6.1
 |===
 
+[[sub:shared-readonly-cache]]
+=== Sharing the dependency cache with other Gradle instances
+
+Instead of <<sub:cache_copy,copying the dependency cache into each container>>, it's possible to mount a shared, read-only folder that will act as a dependency cache for all containers.
+This cache, unlike the classical dependency cache, is accessed without locking, making it possible for multiple builds to read from the cache concurrently.
+When Gradle will look for dependencies (artifacts or metadata), it will look into the classic writable cache, but will also search into the read-only cache.
+It means that if an artifact is present in the read-only cache, it will not be downloaded, but if it's missing from the read-only dependency cache, it will be added to the _local_, writable cache of the Gradle instance.
+In practice it means that the writable cache will contain the _diff_ between what the build requires and what is available in the read-only cache.
+
+NOTE: The shared read-only dependency cache is an incubating feature.
+
+The first step in using the shared dependency cache is actually to create a copy of an existing cache.
+For this you need to follow the <<sub:cache_copy,instructions above>>.
+
+Then set the `GRADLE_RO_DEP_CACHE` environment variable to point to the directory containing the cache:
+
+----
+$GRADLE_RO_DEP_CACHE
+   |-- modules-2 : the read-only dependency cache, should be mounted with read-only privileges
+
+$GRADLE_HOME
+   |-- caches
+         |-- modules-2 : the container specific dependency cache, should be writable
+         |-- ...
+   |-- ...
+----
+
+In a CI environment, it's a good idea to have one build which "seeds" a Gradle dependency cache, which is then _copied_ to a different directory.
+This directory can then be used as the read-only cache for other builds.
+You shouldn't use an existing Gradle installation cache as the read-only cache, because this directory may contain locks and be accessed by the seeding build.
```

Comment:
```suggestion
You shouldn't use an existing Gradle installation cache as the read-only cache, because this directory may contain locks and may be modified by the seeding build.
```